### DISCUSSION TITLE
change output filename

### DISCUSSION THREAD

**Discussion by RA5Dgz2L46** [CineEncoder/cine-encoder#10] [Ideas]
When using release files the output file always ends with "_encoded_XXX". Without compiling the code myself is there a way to change the filename output?

**Comment by CineEncoder** [comment]
Hi!
I will add this option in the settings in the next release.
I think to make a choice: the creation time "_hhmmss_daymonthyear" or choose the prefix "_someprefix" plus the counter " _XXXX".

**Comment by CineEncoder** [comment]
Hi!
In the version 3.3 I added this feature.

**Reply by RA5Dgz2L46** [reply]
Thanks!
